pkg/controller/gardenletdeployer/actuator.go

```diff
@@ -622,6 +624,20 @@ func PrepareGardenletChartValues(
 	// Set the seed name
 	gardenletConfig.SeedConfig.Name = obj.GetName()
 
+	// Set network policy label
+	isGarden, err := gardenletutils.SeedIsGarden(ctx, gardenClient)
```

Comment:
Shouldn't we use the target client here? If the target cluster (where `gardenlet` gets deployed to) is the garden, we need the label.

Reply:
Nice catch, makes sense.